hack/test-e2e.sh

```diff
@@ -0,0 +1,59 @@
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+NSX_OPERATOR_DIR='/root/nsx-operator'
+NSX_OPERATOR_CI_DIR='/root/nsx-operator-ci'
+rm -fr $NSX_OPERATOR_DIR $NSX_OPERATOR_CI_DIR
+git clone https://github.com/vmware-tanzu/nsx-operator/ $NSX_OPERATOR_DIR
+cd $NSX_OPERATOR_DIR
+export GO111MODULE=on
+export GOPATH=/root/go
+export GOROOT=/usr/local/go
+export PATH=$GOROOT/bin:$PATH
+go version
+go env
+make build
+
+# if manual trigger, use the default email and author
+if [ -z ${ghprbActualCommitAuthorEmail+x} ]; then
+    ghprbActualCommitAuthorEmail="manually_trigger@vmware.com"
+fi
+if [ -z ${ghprbActualCommitAuthor+x} ]; then
+    ghprbActualCommitAuthor="manually_trigger"
+fi
+git config user.email ${ghprbActualCommitAuthorEmail}
+git config user.name ${ghprbActualCommitAuthor}
+git remote add pr $ghprbAuthorRepoGitUrl
+git fetch pr $ghprbSourceBranch:$ghprbSourceBranch
+git checkout $ghprbSourceBranch
+
+#kubectl rollout restart deployment nsx-ncp -n vmware-system-nsx
+kubectl scale deployment nsx-ncp -n vmware-system-nsx --replicas=0
+sleep 10
```

Comment:
Do we need to sleep longer in order to wait pod up.

Reply:
`--replicas=0` is used to delete the old Pod, and 10s are enough to delete the Pod during testing in the real testbed.